machine/v1/control-plane-machine-set.aws.stable.testsuite.yaml

```diff
@@ -0,0 +1,73 @@
+apiVersion: apiextensions.k8s.io/v1 # Hack because controller-gen complains if we don't have this
```

Comment:
Grab someone on the machine API team to review please and ask them to imagine dozens of validation tests in this file.

Reply:
I've split the MAPI parts of this into #1297 which I'll merge separately once this is in. Got a few more bits I want to explore there